third_party/xla_client/pjrt_computation_client.h

```diff
@@ -75,6 +75,8 @@ class PjRtComputationClient : public ComputationClient {
 
   void PrepareToExit() override { return; };
 
+  void WaitDeviceExections(const std::vector<std::string>& devices) override;
```

Comment:
Maybe document that if `devices` is empty, it waits for all devices.

Reply:
+1 or make this an optional, where a null optional means "wait for all devices"

Reply:
I will leave a comment, this is align with current `waitdeviceops` behaviors, check https://github.com/pytorch/xla/blob/master/torch_xla/csrc/xla_graph_executor.cpp#L373 and https://github.com/pytorch/xla/blob/master/torch_xla/core/xla_model.py#L965